app/models/loan_document.rb

```diff
@@ -1,9 +1,22 @@
 # frozen_string_literal: true
 
-# ActiveRecord model for loan_documents table
+# == Schema Information
+#
+# Table name: loan_documents
+#
+#  id          :bigint           not null, primary key
+#  loan_id     :bigint
+#  document_id :bigint
+#  approved    :boolean          default(FALSE)
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
 class LoanDocument < ApplicationRecord
   belongs_to :loan
   belongs_to :document
 
+  has_one_attached :file
+
   validates :approved, inclusion: [true, false]
+  validates :file, attached: true
```

Comment:
@ericksonmc is this a presence validation? cause the loan document is initialized without the file, so it would fail in that scenario.